Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md

```diff
@@ -57,18 +70,18 @@ $ direnv reload
 
 1. In this step, we will set up a new virtual enviornment for our new python interpreter to specifically work with Oppia.
 
-    Use the command given below to set up a virtualenv with python `3.8.12`. Be sure to replace the `{unique-virtualenv-name}` with a suitable virtualenv name:
+    Use the command given below to set up a virtualenv with python `3.8.15`. Be sure to replace the `{unique-virtualenv-name}` with a suitable virtualenv name:
 
     ```
-    pyenv virtualenv 3.8.12 {unique-virtualenv-name}
+    pyenv virtualenv 3.8.15 {unique-virtualenv-name}
```

Comment:
Shouldn't this now use direnv?

Reply:
Yeah, I think I can also remove this part, but I feel like it is better to preserve it if there are people that still use just pyenv.